clearml-server-cloud-ready/templates/network-policy-webserver.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "clearml.fullname" . }}-webserver
+  labels:
+    {{- include "clearml.labels" . | nindent 4 }}
+spec:
+  egress:
+  - ports:
+    - port: 8008
+      protocol: TCP
+    to:
+    - podSelector:
```

Comment:
The webserver also acts as a reverse proxy for the fileserver (and in the future might also act as a reverse proxy to the agent services service)

Reply:
Sorry, I didn't catch what you mean here, could you please clarify? I'm not a profi in ClearML, I just deployed it once.

Reply:
Sure :) What I meant was that the webserver service (specifically the nginx it runs) can also redirect requests to the fileserver or to the agent services.